### DISCUSSION TITLE
Stellarium crashing on my laptop

### DISCUSSION THREAD

**Discussion by Bloodfreeze01** [Stellarium/stellarium#2182] [Q&A]
So I need Stellarium for a class that I am in, but the issue is every time I open the program, it turns into the photo. I don't know how to fix it, but I have a new Galaxy book 360. It has the newest windows system as well as a 64 bit. My teacher also doesn't know why its happening and she says that the version on the website doesn't work. Any thoughts? 

Thank you 

![image0](https://user-images.githubusercontent.com/97860606/149687404-cd57f223-9f4d-48ca-b3e1-21bbd2270668.jpeg)

**Comment by gzotti** [comment]
Ugly. Yet another Intel problem? Did you try the known alternative graphic systems, ANGLE mode or MESA mode?

**Reply by Bloodfreeze01** [reply]
Yeah, I've tried both the ANGLE mode and the MESA mode. Both coming with the same outcomes

**Reply by gzotti** [reply]
Do other OpenGL graphics programs work? E.g. Blender? I cannot give further advice, I must assume this is an Intel driver issue that should hopefully be fixed soon.
